BD-4242 Updates for Global Control Groups docs (#9183)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/testing/global_control_group.md

At this point, Braze generates a randomly selected group of users to comprise the selected percentage of your total user base. When saved, all currently active and future campaigns and Canvases no longer send to users in this group, except for campaigns or Canvases that contain any of the tags in your exclusion settings.

## Making changes to your Global Control Group

> The only way to make any changes to your Global Control Group is to disable it and create a new one. For example, if you set up a Global Control Group that is 10% of your audience and you want to decrease its size to 5%, you must disable your current Global Control Group and re-enable a new Global Control Group. 

Reviewer: ```suggestion
You can only make changes to your Global Control Group by disabling it and creating a new one. For example, if you set up a Global Control Group that is 10% of your audience and you want to decrease its size to 5%, you must disable your current Global Control Group and re-enable a new Global Control Group. 
```
